### ISSUE TITLE
keyboard input doesn't work sometimes after loading sliderust presentation

### ISSUE THREAD

**Issue by metajack** [servo/servo#5082]
After loading https://github.com/metajack/sliderust/blob/master/applicative2015.html in servo with `./mach run --release -i --resolution 1280x1024 /path/to/sliderust/applicative2015.html` the keyboard navigation doesn't work about 75% of the time. If I quit and restart enough times, eventually I will get a run that works normally.

**Comment by mskrzypkows** [comment]
It seems that key events are fired and appropriate class is set (element.rs::SetClassName is called, checked with logs). Maybe some CSS style refresh method has to be called.

**Comment by mrobinson** [comment]
I don't see this any longer.
